handbook/ldap/ldap.md

```diff
@@ -0,0 +1,118 @@
+# Sign on with LDAP
+
+You can configure SystemLink to use the Lightweight Directory Access Protocol (LDAP) for user authentication. LDAP attributes and groups can be used to map users to roles and workspaces in SystemLink's role based access control system.
+
+## Assumptions and Prerequisites
+
+- A server running SystemLink. Refer to [Installing and Configuring SystemLink Server and Clients](https://www.ni.com/documentation/en/systemlink/latest/setup/configuring-systemlink-server-clients/) for the basics of setting up a SystemLink server.
+
+- Administrator desktop access to the SystemLink server.
+
+- An LDAP server accessible to the SystemLink server.
+
+- A bind user and bind password for the LDAP server.
+
+- Familiarity with the LDAP attributes available to your organization. If you do not know what LDAP attributes and groups are available to you please contact the system administrator for your LDAP server.
+
+## Enabling LDAP in SystemLink
+
+### Configuring Authentication
+
```

Comment:
Do you think we should have a little intro sentence for every workflow in the ops handbook? It's just a kneejerk reaction for me due to DITA, but in this context I'm fine skipping it on some of the more straightforward workflows.

Reply:
I think I want to leave it out. I feel like all I would add is "Use the following steps to <same things as section title>"